batchflow/models/torch/base.py

```diff
@@ -1172,30 +1172,33 @@ def predict(self, *args, targets=None, feed_dict=None, fetches=None, use_lock=Tr
         """
         inputs, targets = self._make_prediction_inputs(*args, targets=targets, feed_dict=feed_dict, **kwargs)
 
-        if use_lock:
-            self.model_lock.acquire()
+        # Acquire lock, release anyway
+        try:
```

Comment:
`try .. except` block might be extremely slow, which is not we want for each iteration.